tests/test_modeling_tf_pegasus.py

```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-# Copyright 2020 HuggingFace Inc. team.
+# Copyright 2021, Google Inc. and The HuggingFace Inc. team. All rights reserved.
```

Comment:
As Sylvain likes to ask, did the Google authors really contribute to this test file?